src/main/resources/templates/viewResults.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Survey results</title>
+</head>
+<body>
+    <h1>Result of your survey</h1>
+    <!--answers needed for result to show up-->
+    <p th:each="question : ${survey.questions}">
```

Comment:
Is this used anywhere? I couldn't find any place it was, but it seems to me that this won't show the question name only the results.